tests/test_utils.py

```diff
@@ -247,13 +246,11 @@ def test_summary_statistics_are_indentical_to_hardies_paper_confirming_correct_a
     # see http://brucehardie.com/papers/rfm_clv_2005-02-16.pdf
     # RFM and CLV: Using Iso-value Curves for Customer Base Analysis
     df = pd.read_csv('lifetimes/datasets/CDNOW_sample.txt', sep='\s+', header=None, names=['_id', 'id', 'date', 'cds_bought', 'spent'])
-    if sys.version_info[0] < 3:
-        df['date'] = pd.to_datetime(df['date'].astype(unicode))
-    else:
-        df['date'] = pd.to_datetime(df['date'], format='%Y%m%d')
+    df['date'] = pd.to_datetime(df['date'], format='%Y%m%d')
```

Comment:
Because that working both for python2 and python3 without extra if/else statements. To make code cleaner. Not sure is .astype(unicode)` necessary for python2.